Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,9 +154,12 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
-        while line = read.gets
-          puts line if verbose
-          @output += line
+        while buf = read.read(32)
+          if verbose
+            print buf
+            $stdout.flush
+          end
+          @output << buf
```

Comment:
The original `read.gets` will only return for every new line. When we print dots, they will be all in one single line, hence won't be printed out.